test/micro/org/openjdk/bench/java/lang/invoke/LookupAcquire.java

```diff
@@ -57,16 +59,6 @@ public void setup() {
         cached = MethodHandles.lookup();
```

Comment:
Since the benchmark using the cached object is removed, should this field be removed as well?

Reply:
Fixed!